tvtk/pyface/picker.py

```diff
@@ -29,6 +29,7 @@
 from tvtk.common import configure_input
 from apptools.persistence import state_pickler
 
+VTK_MAJOR_VERSION = tvtk.Version().vtk_major_version
```

Comment:
You can just use `from tvtk.common import vtk_major_version` to get this instead of doing this again.